.github/workflows/release.yml

```diff
@@ -273,12 +217,11 @@ jobs:
       - name: Upload release assets
         if: github.event_name == 'release'
         run: |
-          hub release edit `
-            -a ./swagger.json `
-            -a ./docker-stub.zip `
-            -m "Version $($env:RELEASE_VERSION)" `
-            $env:RELEASE_TAG_NAME
-        shell: pwsh
+          hub release edit \
+            -a ./swagger.json \
+            -a ./docker-stub.zip \
+            -m "Version $RELEASE_VERSION" \
+            $RELEASE_TAG_NAME
```

Comment:
With shell not defined does it by default use bash?

Reply:
yep